crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -59,6 +60,25 @@ impl<F: Send + Clone> GetTracker<F, PaymentData<F>, api::PaymentsRequest> for Pa
             .await
             .to_not_found_response(errors::ApiErrorResponse::PaymentNotFound)?;
 
+        if auth_flow == services::AuthFlow::Client && request.customer_id.is_some() {
+            let response = request
```

Comment:
An appropriate name for this variable would be `is_same_customer`

Reply:
done!

Reply:
My bad, since this is checking whether the customer is not same, it should be `is_not_same_customer`. Can you please correct it?

Reply:
done!